client/package.json

```diff
@@ -2,8 +2,10 @@
   "name": "@kyma-project/luigi-client",
```

Comment:
Please change the name of this package. It won't be the `@kyma-project/luigi-client` package anymore, the transpiled  version will be.